### DISCUSSION TITLE
Incorrect redirection for `Accept-Language` header

### DISCUSSION THREAD

**Discussion by dector** [mdn/mdn-community#225] [Platform] [answered]
**STR:**
- Setup `Accept-Language: en-US,en;q=0.7,uk;q=0.3` header (e.g. use English and Ukrainian locales in setting).
- Open `https://developer.mozilla.org/`.

**ER:**
- Should be redirected to `https://developer.mozilla.org/en-US/`.

**AR:**
- Receiving redirect to `https://developer.mozilla.org/ru/`.

_Please, don't enforce occupation imperialistic culture in my country. It murders people._

Thanks!

With :heart: to Mozilla from 🇺🇦 !

**Comment by caugner** [comment, marked as answer]
Hi @dector,

I cannot reproduce the issue:

```
% curl -I --header 'Accept-Language: en-US,en;q=0.7,uk;q=0.3' https://developer.mozilla.org/
HTTP/2 302 
content-length: 0
server: CloudFront
date: Mon, 05 Sep 2022 12:13:34 GMT
location: /en-US/
cache-control: no-store
x-cache: LambdaGeneratedResponse from cloudfront
via: 1.1 1bb7dd4aad75e0709d676cb31c440fee.cloudfront.net (CloudFront)
x-amz-cf-pop: CDG50-C2
x-amz-cf-id: k1fvsuozNojWCPAwXYNsZNK2umqWjuXsZxNOLVOD_vHwOI6uCbuBSg==
```

See also: https://github.com/mdn/yari/issues/6309
